packages/experimental/Menu/src/MenuList/MenuList.tsx

```diff
@@ -33,18 +35,53 @@ export const MenuList = compose<MenuListType>({
   ...stylingSettings,
   slots: {
     root: MenuStack,
+    focusZone: FocusZone,
   },
   useRender: (userProps: MenuListProps, useSlots: UseSlots<MenuListType>) => {
     const menuList = useMenuList(userProps);
     const contextValue = useMenuListContextValue(menuList);
     const Slots = useSlots(menuList);
 
+    /**
+     * On macOS, focus isn't placed by default on the first focusable element. We get around this by focusing on the inner FocusZone
+     * hosting the menu. For whatever reason, to get the timing _just_ right to actually focus, we need an additional `setTimeout`
+     *  on top of the `useLayoutEffect` hook.
+     */
+    const focusZoneRef = React.useRef<IFocusable>(null);
+
+    React.useLayoutEffect(() => {
+      if (Platform.OS === 'macos') {
+        setTimeout(() => {
+          focusZoneRef.current?.focus();
+        }, 0);
+      }
+    });
+
     return (_final: MenuListProps, children: React.ReactNode) => {
-      return (
-        <MenuListProvider value={contextValue}>
-          <Slots.root>{children}</Slots.root>
-        </MenuListProvider>
-      );
+      if (Platform.OS === 'macos') {
+        return (
+          <MenuListProvider value={contextValue}>
+            <Slots.root>
+              <Slots.focusZone
+                // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+                // @ts-ignore
```

Comment:
Can we add a comment on why this is needed

Reply:
Sure. On macOS, FocusZone inherits from all of ViewProps because natively, `FRNFocusZone` is a subclass of `RCTView` (the native class that backs `<View>`). As such, we can set `enableFocusRing` on it. I think on win32, FocusZone does _not_ inherit all of View props, and thus I didn't extend FocusZoneProps to include all of ViewProps. 

Something to revisit if we ever do FocusZoneV1 I suppose. I'll update the comment here.

Reply:
Updated. An aside: It seems even win32 FocusZone should inherit ViewProps natively so we can perhaps change the type. Feels out of scope for this PR so I'll address that separately.